pkg/account/domain/account.go

```diff
@@ -305,6 +305,19 @@ func (a *AccountV2) resetDefaultFilter(targetFilter proto.FilterTargetType, envi
 	}
 }
 
+func (a *AccountV2) GetAccountFullName() string {
+	if a == nil {
+		return ""
+	}
```

Comment:
It can't be nil here.

```suggestion
```

Reply:
I have fixed that and added test cases for the method